dashboard/app/models/frequency_analysis.rb

```diff
@@ -0,0 +1,13 @@
+class FrequencyAnalysis < Widget
```

Comment:
Hmm actually I don't know if this is true when no users will have progress on the new level.  @laurelfan is this different than the Pixelation model because no users have FrequencyAnalysis progress?